api/Request.json

```diff
@@ -0,0 +1,1425 @@
+{
+  "api": {
+    "Request": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/Request",
+        "support": {
+          "webview_android": {
+            "version_added": "42"
+          },
+          "chrome": {
+            "version_added": "42"
+          },
+          "chrome_android": {
+            "version_added": "42"
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": "39"
+          },
+          "firefox_android": {
+            "version_added": true
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": "28"
+          },
+          "opera_android": {
+            "version_added": "28"
+          },
+          "safari": {
+            "version_added": false
+          },
+          "safari_ios": {
+            "version_added": false
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "request_body_formdata": {
```

Comment:
Remove this, the data is here https://github.com/mdn/browser-compat-data/blob/master/api/Body.json#L380

Reply:
👍